genesyscloud/auth_division/genesyscloud_auth_division_proxy.go

```diff
@@ -131,13 +131,15 @@ func getAuthDivisionByIdFn(ctx context.Context, p *authDivisionProxy, id string,
 }
 
 func getAuthDivisionIdByNameFn(ctx context.Context, p *authDivisionProxy, name string) (string, *platformclientv2.APIResponse, bool, error) {
+	notFoundError := fmt.Errorf("unable to find auth division with name %s", name)
+
 	authzDivisions, resp, err := getAllAuthDivisionFn(ctx, p, name)
 	if err != nil {
 		return "", resp, false, err
 	}
 
 	if authzDivisions == nil || len(*authzDivisions) == 0 {
-		return "", resp, true, err
+		return "", resp, true, notFoundError
```

Comment:
Because we were returning nil as the error here, there was no error info displayed when no division was found with that name. It would just say `Error requesting division {{name}} | error: <nil>`